dotcom-rendering/src/amp/lib/find-adslots.ts

```diff
@@ -84,8 +84,9 @@ const hasForwardBuffer = (
 
 	const enoughCharsForward = meetsThreshold || noForwardsEmbeds;
 
-	const neighbourSuitable = elements[index]
-		? suitableAdNeighbour(elements[index])
+	const neighbour = elements[index];
+	const neighbourSuitable = neighbour
+		? suitableAdNeighbour(neighbour)
```

Comment:
Typescript needs the `neighbour` extracted to know that it is defined in the left side of the ternary.